src/main.js

```diff
@@ -154,85 +154,65 @@ SearchResults.prototype.search = async function search() {
-                </li>
-            `
-          )
-          .join("")}
+      .map(
+        (result) => {
+          const caseUrl = result.case.caseType === "DIGITAL_LPA"
+            ? `/lpa/frontend/lpa/${result.case.uId}`
+            : result.case.caseType.toLowerCase() === "order" 
+            ? `/supervision/#/clients/${result.id}?order=${result.case.id}`
+            : `/lpa/person/${result.id}/${result.case.id}`;
+  
+          return `
+                  <li class="${CLASSES.item}">
+                   <a class="govuk-!-font-weight-bold govuk-link" href="${caseUrl}">
+                      ${escapeHTML(result.firstname)} ${escapeHTML(result.surname)}, ${result.personType}
+                    </a>
+                      <p class="${CLASSES.link}">
+                          <a class="govuk-link" href="${caseUrl}">
+                            ${result.case.uId}
+                          </a>
+                      </p>
+                      <dl class="govuk-summary-list govuk-summary-list--no-border govuk-!-margin-bottom-0">
+                          <div class="govuk-summary-list__row">
+                              <dt class="govuk-summary-list__key govuk-!-padding-top-1 govuk-!-padding-bottom-1"><abbr title="Date of birth">DOB:</abbr></dt>
+                              <dd class="govuk-summary-list__value govuk-!-padding-top-1 govuk-!-padding-bottom-1">${result.dob}</dd>
+                          </div>
+                          <div class="govuk-summary-list__row">
+                              <dt class="govuk-summary-list__key govuk-!-padding-top-1 govuk-!-padding-bottom-1">Address:</dt>
+                              <dd class="govuk-summary-list__value govuk-!-padding-top-1 govuk-!-padding-bottom-1">${
+                              result.addresses ? formatAddress(result.addresses[0]) : ""
+                              }</dd>
+                          </div>
+                          ${result.case.status ? `
+                          <div class="govuk-summary-list__row">
+                              <dt class="govuk-summary-list__key govuk-!-padding-top-1 govuk-!-padding-bottom-1">Status:</dt>
+                              <dd class="govuk-summary-list__value govuk-!-padding-top-1 govuk-!-padding-bottom-1">
+                                <strong class="govuk-tag govuk-tag--${statusColour(result.case.status)}">
+                                  ${result.case.status}
+                                </strong>
+                              </dd>
+                          </div>
+                          ` : ""}
+                          <div class="govuk-summary-list__row">
+                              <dt class="govuk-summary-list__key govuk-!-padding-top-1 govuk-!-padding-bottom-1">Type:</dt>
+                              <dd class="govuk-summary-list__value govuk-!-padding-top-1 govuk-!-padding-bottom-1">${
+                                  result.case.caseType.toUpperCase() === "DIGITAL_LPA"
```

Comment:
I liked that you moved your logic to a variable do you think that you could do the same here?

I know this was not you, I am 99% sure it was me but I did not think of the way you did yours. 

No issue if not I will still approve and come back to clean up my work 👍